### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by danielochuba**
# BlogApp: Add forms
### In this pull request I implemented the following:
- add post with current user
- add comment with current user
- add like with current user

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
-<li>Blog post</li>
-<li>Number of comments</li>
-<li>Number of likes</li>
-<li>All comments</li>
-</ul>
\ No newline at end of file
+<section class="user-post">
+    <div class="post-content">
+        <% user = User.find(@user_post.author_id) %>
+        <p class="post-stats"><span>Post <%= @user_post.id %> by <strong><%= user.name %></strong> </span><span>Comments: <%= @user_post.comments_counter %>, Likes: <%= @user_post.likes_counter %></span></p>
```
Comment (codepantha) [inline app/views/posts/show.html.erb:4]: - [ ] You already have the `create` action in the likes controller and setup the route for liking a post, however, a user still needs to be able to like a post by clicking either a link or a button as you can see in the requirement below:
image


### HINT 🔔 
You can create a button that triggers the like action like this 👇 
```erb
<%= button_to "Like", post_likes_path(user_id: current_user, post_id: @post.id), method: :post, remote: true %>
```

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @danielochuba 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Application controller contains the `#current_user` method 📌 
✔️ Added forms for creating posts and comments 📌 
✔️ No linter errors 🟢 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by danielochuba** [comment]
@codepantha please can you help me through am finding this very difficult here...

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @danielochuba, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did well to fix the like button functionality.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**